### PR TITLE
Improvements for simplelink academy conversion script

### PR DESCRIPTION
**Pull request by biagiom**
Hi @jcormier,
I have updated the **convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh** conversion script
for the simplelink academy package, improving documentation and removing useless code; in fact, according to the [log file of simplelink academy conversion script](https://github.com/jcormier/TI_BLE_CC2650_Linux_Convert/blob/master/convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh.log), sed returns "_no input files_"
Could you check if these changes work for you ?

Best regards.

### CODE REVIEW

File: convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh [hunk 1]
```diff
@@ -25,14 +26,17 @@ fi
...
 TIRTOS_VERSION=tirtos_cc13xx_cc26xx_2_20_01_08
 XDCTOOLS_VERSION=xdctools_3_32_00_06_core
 XDCTOOLS_SOURCE=http://software-dl.ti.com/dsps/dsps_public_sw/sdo_sb/targetcontent/rtsc/3_32_00_06/index_FDS.html
 
 WINE_ROOT=~/.wine/drive_c/ti
 
-BLE_SDK_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink_academy_${BLE_SDK_VERSION}/modules
+SIMPLELINK_SDK_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink_academy_${SIMPLELINK_SDK_VERSION}
+BLE_SDK_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink/${BLE_SDK_VERSION}
+SIMPLELINK_SDK_MODULES_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink_academy_${SIMPLELINK_SDK_VERSION}/modules
```
Comment (jcormier) [inline convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh]: Shouldn't this line use ${SIMPLELINK_SDK_DIRECTORY}

**Review by jcormier — COMMENTED** [review]
Looks like a nice cleanup.  Thanks.

File: convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh (hunk 1 above)
Comment (biagiom) [reply convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh]: Thanks for your review.
I have added a new commit that fix the SIMPLELINK_SDK_MODULES_DIRECTORY variable

**Comment by jcormier** [comment]
LGTM. Thanks

**Comment by biagiom** [comment]
You are welcome!

**Pull request merged**